using_images/other_images/jenkins.adoc

```diff
@@ -52,9 +52,24 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+ifdef::openshift-origin[]
```

Comment:
I ifdef'ed cause I thought that is what was required until 3.4 GA'ed.  I am not allowed to add labels to the PR, but will remove the ifdef.